lib/disconnected_pulp.rb

```diff
@@ -134,16 +134,30 @@ def configure(remove_disabled = false, puppet = false, puppet_forge_url, puppet_
         repo = active_repos[repoid]
         relative_url = URI.split(repo.url)[5]
 
-        # Yum, ISO and Export
-        distributors = [Runcible::Models::YumDistributor.new(relative_url, true, false, {:id => 'yum_distributor'}),
-            Runcible::Models::ExportDistributor.new(true, false)]
+        if repoid.include? "-files-"
```

Comment:
Have a look at how `katello-disconnected info` works, the repo type is stored in the imported manifest.